nuxeo-user-group-management/nuxeo-edit-password.html

```diff
@@ -58,18 +57,30 @@
               type: String,
               notify: true,
             },
+            _confirmationPassword: {
+              type: String,
+            },
           };
         }
 
+        _computeErrorMessage(password) {
+          if (!password) {
+            return this.i18n('editPassword.required');
+          } else {
+            return this.i18n('editPassword.noMatch');
+          }
+        }
+
         _getValidity() {
-          const v1 = this.$.password.value;
-          const v2 = this.$.passwordConfirmation.value;
-          return v1 && v2 && v1 === v2;
+          if (typeof this.password !== 'string') {
+            return false;
+          }
+          return this.password === this._confirmationPassword && this.password.length > 0;
```

Comment:
This logic is already done by the `nuxeo-input`, since we're using a `pattern` here. This circumvents input validation, we will fail to display feedback to the user in a couple of scenarios, with the most notable being when the user types a password and hits the create button, without inputting any password confirmation. Instead, `_getValidity` should just be:
```
        _getValidity() {
          return this.$.passwordConfirmation.validate();
        }
```